### DISCUSSION TITLE
Replace a part of a 3d glb model with a new part loaded from local path failed

### DISCUSSION THREAD

**Discussion by whereisnothing** [google/filament#5993] [General]
**Describe the bug**
Could you please give me a demo on how to replace a part (eg.  hair) of a 3D glb model with a new part (eg. hair) loaded from 
 a local file path?I tried many apis,but can't succeed(What I want to achieve is as follows:Replace the original hair of a 3D glb model with a new hair. But the result is: The new hair's position is wrong and I can't successfully use attachSkin api to blend the new hair into the skinning system of the original 3D glb model).You can see the video I uploaded to know the problem I'm facing now.Hope your reply.Thanks for your time.

**To Reproduce**

**Expected behavior**
Can remove an old hair from 3d glb model and add a new hair  to the 3d glb model ,and the new hair can be blended into the original 3d glb model perfectly.

**Screenshots**

https://user-images.githubusercontent.com/6499374/186361888-a0e4edc1-774d-4df4-bbb4-f9a2a271ce0e.mp4

When only load the new glb model(only contains hair of a whole 3d glb model) and call scene.addEntity and call asset.attachSkin,
the new hair malposition and display abnormally.You can see video as follows.

https://user-images.githubusercontent.com/6499374/186363497-95901f62-83de-4ddc-8c59-f995a015d9bf.mp4

**Logs**
If applicable, copy logs from your console here. Please *do not*
use screenshots of logs, copy them as text.

**Desktop (please complete the following information):**
 - GPU: [e.g. NVIDIA GTX 1080]
 - Backend: [OpenGL/Vulkan]

**Smartphone (please complete the following information):**
 - Device: Huawei Honor
 - OS: Android 10

**Additional context**

**Comment by dandingol03** [comment]
any progress?
